src/Engines/nativeEngine.ts

```diff
@@ -1503,7 +1503,9 @@ export class NativeEngine extends Engine {
     }
 
     protected _deleteTexture(texture: Nullable<WebGLTexture>): void {
-        this._native.deleteTexture(texture);
+        if (texture) {

```

Comment:
This check probably should be where _deleteTexture is called in thinEngine and this function shouldn't take a Nullable.